internal/schema/schema.go

```diff
@@ -106,6 +106,15 @@ func (s Schema) Hash() (string, error) {
 	return fmt.Sprintf("%x", hashVal), nil
 }
 
+type ReplicaIdentity string
+
+const (
+	ReplicaIdentityDefault ReplicaIdentity = "d"
+	ReplicaIdentityNothing ReplicaIdentity = "n"
```

Comment:
I chose to not do any sort of enum conversion in the schema extraction. This will enable us to support "new replica identities" added by Postgres assuming there is no diff in the identity. Otherwise, if postgres added some new replica identity, pg-schema-diff would always error out on it even if there was no diff